src/modules/gcloud.rs

```diff
@@ -89,6 +89,10 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     }
 
     let (config_name, config_path) = get_current_config(context)?;
+
+    if config_name == "NONE" {
+        return None;
+    }
```

Comment:
I think I would prefer putting this into `get_current_config`, but this is fine too.